packages/devtools_testing/pubspec.yaml

```diff
@@ -0,0 +1,9 @@
+name: devtools_testing
+version: 1.0.0-dev
+environment:
+  sdk: ^2.2.0
+dependencies:
```

Comment:
add a comment with dependency overrides to use a relative import for package:devtools

Reply:
had to add a real override to resolve the merge conflict anyways :D